static/js/TopicPage.jsx

```diff
@@ -351,13 +352,30 @@ const TopicHeader = ({ topic, topicData, multiPanel, isCat, setNavTopic, openDis
         <div>
           <div className="sectionTitleText authorIndexTitle"><InterfaceText>Works on Sefaria</InterfaceText></div>
           <div className="authorIndexList">
-            {topicData.indexes.map(({text, url}) => <SimpleLinkedBlock key={url} {...text} url={url} classes="authorIndex" />)}
+            {topicData.indexes.map(({url, title, description}) => <AuthorDisplayItem key={url} url={url} enTitle={title.en} heTitle={title.he} enDesc={description.en} heDesc={description.he}/>)}
           </div>
         </div>
        : null}
     </div>
 );}
 
+const AuthorDisplayItem = ({url, enTitle, heTitle, enDesc, heDesc}) => {
+  const classes = classNames({ navBlockTitle: 1 });
```

Comment:
`classNames` is useful when there are classes that need to change based on a variable. Since `navBlockTitle` is constant, you can add it directly to the `className` prop of the `<a>` tag.